setup.py

```diff
@@ -2,13 +2,16 @@
 # -*- coding: utf-8 -*-
 
 from setuptools import setup, find_packages
+import scvi
```

Comment:
I'm not sure we can import scvi here. This is the script that runs _to install_ scvi. If it imports scvi, that means scvi needs to already be installed.   Are there other examples of `setup.py` files that import the package they are installing?

Reply:
Sorry you're right 😊 That might cause a problem. I found another way of reading `__version__` from `scvi.__init__.py` and I just pushed the code with the new solution.